cmd/handler-utils.go

```diff
@@ -498,7 +498,6 @@ func proxyRequest(ctx context.Context, w http.ResponseWriter, r *http.Request, e
 		RoundTripper: ep.Transport,
 		ErrorHandler: func(w http.ResponseWriter, r *http.Request, err error) {
 			success = false
-			w.WriteHeader(http.StatusBadGateway)
 		},
 		Logger: func(err error) {
 			logger.LogIf(GlobalContext, err)
```

Comment:
This logger is not used and useless either we remove it or add it to ErrorHandler such as 

```
if err != nil && !error.Is(err, context.Canceled) {
     logger.LogIf(GlobalContext, err)
}
```